.github/workflows/plugins-jdk17-test.1.yaml

```diff
@@ -63,6 +63,7 @@ jobs:
           - c3p0-0.9.0.x-0.9.1.x-scenario
           - c3p0-0.9.2.x-0.10.x-scenario
           - spring-scheduled-6.x-scenario
+          - caffeine-3.x-scenario
```

Comment:
Is there any reason running this in JDK17?

Reply:
Because I chose to use springboot3.x, no other special reason.